app/src/test/java/edu/uc/brown5a7/BottomsUp/DrinkDTOMockkUnitTest.kt

```diff
@@ -98,35 +109,19 @@ class DrinkDTOMockkUnitTest {
         confirmVerified(drinkService)
     }
 
-    private fun whenSearchForMariageFreres() {
-        mvm.fetchDrinks("Mariage Freres")
-    }
-
     private fun thenResultContainsMariageFreres() {
-        var MariageFreresFound = false
+        var mariageFreresFound = false
```

Comment:
Changing variable to stay consistent with camel case.

Reply:
Yes... this is consistent with convention.